app/assets/stylesheets/dashboard/protocol.css.scss

```diff
@@ -69,6 +69,31 @@ textarea {
 
 /* New/Edit Protocol Form End */
 
+/* Show Protocol Begin */
 .view-protocol-details-modal {
   width: 50% !important;
 }
+
+.export-consolidated-request {
+  display: inline-block;
+}
+
+.service-requests-table {
```

Comment:
Some specific styling to get the buttons to display nicely in both normal view and card view